en/micro-integrator/docs/setup/observability/setting-up-cloud-native-observability-in-kubernetes.md

```diff
@@ -1,20 +1,23 @@
-# Setting up the cloud native observability deployment in a Kubernetes environment
+# Setting up Cloud-Native Observability on Kubernetes
 
-To streamline the deployment of the cloud native observability solution in Kubernetes, WSO2 EI provides a Helm chart that you can utilize to deploy the solution to your Kubernetes cluster. The deployment installs the relevant products and adds the required configurations. After the installation, you can directly use the observability solution with a very few additional configurations. 
+Follow the instructions given below to set up a cloud-native observability solution in a Kubernetes environment. 
 
-## Deploying the basic cloud native observability solution
+To streamline the deployment of the cloud native observability solution in Kubernetes, WSO2 EI provides a Helm chart to deploy the solution to your Kubernetes cluster. The deployment installs the relevant products and adds the required configurations. After the installation, you can directly use the observability solution with a very few additional configurations. 
```

Comment:
"To streamline the deployment of the cloud native observability solution in Kubernetes, WSO2 EI provides a Helm chart via which you can deploy the solution to your Kubernetes cluster."

Suggesting this change to avoid the repetition of "to" within the same sentence.